{{cookiecutter.github_project_name}}/src/index.ts

```diff
@@ -0,0 +1,5 @@
+
+
+export * from './example';
+export * from './plugin';
```

Comment:
I had to include this because otherwise the karma test runner would complain that it could not find it.  cf https://github.com/monounity/karma-typescript/issues/79.